project/Dependencies.scala

```diff
@@ -22,6 +22,7 @@ object Dependencies {
   lazy val common = core ++ infoset ++ test
 
   lazy val core = Seq(
+    "com.lihaoyi" %% "os-lib" % "0.7.1", // for generating C source files
```

Comment:
Latest version of 0.7.3. Might as well update now instead of waiting for this to merge then getting a PR from scala steward.